coderd/database/migrations/000168_pg_coord_tailnet_v2_api.up.sql

```diff
@@ -0,0 +1,72 @@
+BEGIN;
+
+CREATE TYPE tailnet_status AS ENUM (
+	'ok',
+	'lost'
+);
+
+CREATE TABLE tailnet_peers (
+	id uuid NOT NULL,
+	coordinator_id uuid NOT NULL,
```

Comment:
Coordinator ID means replica ID? Could this be a foreign key to the replicas table instead?

Reply:
We have a separate table of Coordinators.  They have different lifetimes for what is considered "stale" so I don't want to commingle, even though most of the time it's 1:1 between a replica and coordinator.